### PR TITLE
add colab to readme

### PR DESCRIPTION
**Pull request by anishau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Growing Neural Cellular Automata - Task Experiments  
 
-Based on the original work [Growing Neural Cellular Automata](https://distill.pub/2020/growing-ca/) by Alexander Mordvintsev et al.  
+Based on the original work [Growing Neural Cellular Automata](https://distill.pub/2020/growing-ca/) by Alexander Mordvintsev et al. 
   
 The writing on this page is adapted from posts and discussion in the distill community slack group.  
 
+The [particle simulation](http://transdimensional.xyz/projects/neural_ca/index.html) code is [available as a google colab](https://colab.research.google.com/drive/1XaNCLrVyp5JYXgP_glWExSUyhbV8OaWb?usp=sharing)
```
Comment (PWhiddy) [inline README.md:7]: How about "Thanks to @anishau code to train the [particle simulation] model is available as a [colab notebook]"

Comment (anishau) [reply README.md]: sounds good! I sent it back to ya for the minorest aesthetic thing - my social media brain is used to seeing '@' be a link to somwhere, but git markdown doesn't do that, wasn't sure what the standard was.

**Comment by PWhiddy** [comment]
One final suggestion, you can make this stand out a little more with the "try in colab" badge. 

[Open In Colab](https://colab.research.google.com/drive/1XaNCLrVyp5JYXgP_glWExSUyhbV8OaWb?usp=sharing)

**Comment by PWhiddy** [comment]
It looks like in your latest commit the badge just links to the svg image rather than the notebook itself? I think the example in my previous comment works

**Pull request merged**